zammad/README.md

```diff
@@ -78,6 +70,12 @@ memcached:
   containerSecurityContext:
     enabled: false
 
+minio:
+  podSecurityContext:
```

Comment:
why do we disable podSecurityContext & podSecurityContext?

Reply:
should this config not go to bottom of the page, to the subachrt config section?

Reply:
Please have a look at the context. This change is in the OpenShift section, where I adapted it in the same way like the other entries that were already there. Should be correct IMHO.